tubular/base.py

```diff
@@ -64,6 +66,14 @@ def __init__(
             msg = f"{self.classname()}: verbose must be a bool"
             raise TypeError(msg)
 
+        # Keyword argument `kwarg2` is going to be dropped completely.
+        if type(copy) is int:
```

Comment:
Q: Why test the type of copy? Wouldn't `if copy is not None` be right?

Reply:
Yep you're right, updated it in this commit [here](https://github.com/lvgig/tubular/pull/197/commits/67401d7a1a19d9e4a36902975bfa3bae83deb415)